### PR TITLE
Create README.fr.md

### PR DESCRIPTION
**Pull request by kushal34712**
added french translation for readme.md file

### CODE REVIEW

**Review by SauravMaheshkar — CHANGES_REQUESTED** [review]
Thanks for adding a README in another language. I'd propose a couple of changes:

* Place all translations in `i18n/`. AFAIK this is considered to be the usual place for translations, viz. https://github.com/supabase/supabase/blob/master/i18n/languages.md
* Reference translated versions from the main README as well.

**Review by SauravMaheshkar — CHANGES_REQUESTED** [review]
Could you also update the main README to point to this translated version?

Add a line below the various badges linking the various READMEs. Also make sure to cross link the READMEs with each other.

**Comment by kushal34712** [comment]
i have added the same now.

File: README.md [hunk 1]
```diff
@@ -7,6 +7,12 @@
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 [![Discord](https://img.shields.io/discord/752876370337726585?logo=discord&logoColor=white&label=discord&color=7289da)](https://discord.gg/xvNJW94)
 
+<h2 align="center" style="border-bottom: none">
+
+[**Français**](i18n/README.fr.md)
+
+</h2>
+
```
Comment (SauravMaheshkar) [inline README.md:15]:
```suggestion
<h4 align="center">
  <p>
    <b>English</b> |
    <a href="https://github.com/lightly-ai/lightly/blob/master/i18n/README.fr.md">Français</a>
  </p>
</h4>
```

File: i18n/README.fr.md [hunk 2]
```diff
@@ -0,0 +1,413 @@
+![Logo d'apprentissage auto-supervisé LightlySSL](docs/logos/lightly_SSL_logo_crop.png)
+
+![GitHub](https://img.shields.io/github/license/lightly-ai/lightly)
+![Tests unitaires](https://github.com/lightly-ai/lightly/workflows/Unit%20Tests/badge.svg)
+[![PyPI](https://img.shields.io/pypi/v/lightly)](https://pypi.org/project/lightly/)
+[![Téléchargements](https://static.pepy.tech/badge/lightly)](https://pepy.tech/project/lightly)
+[![Style de code : noir](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+[![Discord](https://img.shields.io/discord/752876370337726585?logo=discord&logoColor=white&label=discord&color=7289da)](https://discord.gg/xvNJW94)
+
```
Comment (SauravMaheshkar) [inline i18n/README.fr.md:9]: And let's add the same here as well, linking back to the English README.

```suggestion
<h4 align="center">
  <p>
    <a href="https://github.com/lightly-ai/lightly/blob/master/README.md">English</a> |
    <b>Français</b>
  </p>
</h4>
```

**Review by SauravMaheshkar — CHANGES_REQUESTED** [review]
We're almost there 😄.

File: i18n/README.fr.md [hunk 3]
```diff
@@ -0,0 +1,413 @@
+![Logo d'apprentissage auto-supervisé LightlySSL](docs/logos/lightly_SSL_logo_crop.png)
+
+![GitHub](https://img.shields.io/github/license/lightly-ai/lightly)
+![Tests unitaires](https://github.com/lightly-ai/lightly/workflows/Unit%20Tests/badge.svg)
+[![PyPI](https://img.shields.io/pypi/v/lightly)](https://pypi.org/project/lightly/)
+[![Téléchargements](https://static.pepy.tech/badge/lightly)](https://pepy.tech/project/lightly)
+[![Style de code : noir](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
```
Comment (guarin) [inline i18n/README.fr.md:7]:
```suggestion
[![Style de code : black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
```

File: README.md [hunk 4]
```diff
@@ -7,6 +7,12 @@
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 [![Discord](https://img.shields.io/discord/752876370337726585?logo=discord&logoColor=white&label=discord&color=7289da)](https://discord.gg/xvNJW94)
 
+<h2 align="center" style="border-bottom: none">
+
+[**Français**](i18n/README.fr.md)
```
Comment (guarin) [inline README.md:12]: I believe the standard is to use an underscore for the filename:
```suggestion
[**Français**](i18n/README_fr.md)
```

File: i18n/README.fr.md [hunk 5]
```diff
@@ -0,0 +1,413 @@
...
+  - Dans tous les cas, ce framework sera toujours libre d'utilisation, même à des fins commerciales.
+
+## Légèrement dans la recherche
+
+- [Apprentissage auto-supervisé par ingénierie inverse, 2023](https://arxiv.org/abs/2305.15614)
+- [Apprentissage des représentations visuelles via un échantillonnage guidé par le langage, 2023](https://arxiv.org/pdf/2302.12248.pdf)
+- [Méthodes d'apprentissage auto-supervisées pour la classification des caries dentaires efficace sur le label, 2022](https://www.mdpi.com/2075-4418/12/5/1237)
+- [DPCL : Apprentissage de représentation contrastive avec confidentialité différentielle, 2022](https://assets.researchsquare.com/files/rs-1516950/v1_covered.pdf?c=1654486158)
+- [Apprentissage contrastif découplé, 2021](https://arxiv.org/abs/2110.06848)
+- [apprentissage en solo : une bibliothèque de méthodes auto-supervisées pour l'apprentissage de la représentation visuelle, 2021](https://www.jmlr.org/papers/volume23/21-1155/21-1155.pdf)
```
Comment (guarin) [inline i18n/README.fr.md:402]: Please do not translate names and paper titles. This also applies for the rest of the document.

File: i18n/README.fr.md [hunk 6]
```diff
@@ -0,0 +1,413 @@
...
+
+- Jetez un œil à notre [chaîne #papers sur Discord](https://discord.com/channels/752876370337726585/815153188487299083)
+  pour les derniers documents d'apprentissage auto-supervisés.
+- [Un livre de recettes d'apprentissage auto-supervisé, 2023](https://arxiv.org/abs/2304.12210)
+- [Les auto-encodeurs masqués sont des apprenants en vision évolutive, 2021](https://arxiv.org/abs/2111.06377)
+- [Propriétés émergentes des transformateurs de vision auto-supervisés, 2021](https://arxiv.org/abs/2104.14294)
+- [Apprentissage non supervisé des fonctionnalités visuelles par affectations de clusters contrastées, 2021](https://arxiv.org/abs/2006.09882)
+- [Ce qui ne devrait pas être contrastif dans l'apprentissage contrastif, 2020](https://arxiv.org/abs/2008.05659)
+- [Un cadre simple pour l'apprentissage contrastif des représentations visuelles, 2020](https://arxiv.org/abs/2002.05709)
+- [Momentum Contrast pour l'apprentissage de la représentation visuelle non supervisé, 2020](https://arxiv.org/abs/1911.05722)
```
Comment (guarin) [inline i18n/README.fr.md:372]: Do not translate paper names

File: i18n/README.fr.md [hunk 7]
```diff
@@ -0,0 +1,413 @@
...
+| BarlowTwins | Rés50 | 256 | 100 | 62,9 | 72,6 | 45.6 | [lien](https://lightly-ssl-checkpoints.s3.amazonaws.com/imagenet_resnet50_barlowtwins_2023-08-18_00-11-03/pretrain/version_0/events.out.tfevents.1692310273.Machine2.569794.0) | [lien](https://lightly-ssl-checkpoints.s3.amazonaws.com/imagenet_resnet50_barlowtwins_2023-08-18_00-11-03/pretrain/version_0/checkpoints/epoch%3D99-step%3D500400.ckpt) |
+| BYOL | Rés50 | 256 | 100 | 62,5 | 74,5 | 46,0 | [lien](https://lightly-ssl-checkpoints.s3.amazonaws.com/imagenet_resnet50_byol_2024-02-14_16-10-09/pretrain/version_0/events.out.tfevents.1707923418.Machine2.3205.0) | [lien](https://lightly-ssl-checkpoints.s3.amazonaws.com/imagenet_resnet50_byol_2024-02-14_16-10-09/pretrain/version_0/checkpoints/epoch%3D99-step%3D500400.ckpt) |
+| DINOSAURE | Rés50 | 128 | 100 | 68.2 | 72,5 | 49,9 | [lien](https://lightly-ssl-checkpoints.s3.amazonaws.com/imagenet_resnet50_dino_2023-06-06_13-59-48/pretrain/version_0/events.out.tfevents.1686052799.Machine2.482599.0) | [lien](https://lightly-ssl-checkpoints.s3.amazonaws.com/imagenet_resnet50_dino_2023-06-06_13-59-48/pretrain/version_0/checkpoints/epoch%3D99-step%3D1000900.ckpt) |
```
Comment (guarin) [inline i18n/README.fr.md:284]: DINOSAURE 😅

**Review by guarin — COMMENTED** [review]
Thanks for the contribution! Readmes in multiple languages will be handy. Left a bunch of comments.

Also just to manage expectations, we won't be able to maintain translated readmes and will have to rely on contributors to keep them up-to-date.

File: i18n/README.fr.md (hunk 7 above)
Comment (kushal34712) [reply i18n/README.fr.md:289]: its translation

File: i18n/README.fr.md (hunk 6 above)
Comment (kushal34712) [reply i18n/README.fr.md:377]: but then how can it be understood by them then

File: i18n/README.fr.md (hunk 5 above)
Comment (kushal34712) [reply i18n/README.fr.md:407]: same applicable here too

**Comment by kushal34712** [comment]
@SauravMaheshkar plz do merge it i have done all the changes

File: i18n/README.fr.md (hunk 6 above)
Comment (guarin) [reply i18n/README.fr.md:377]: We shouldn't modify original titles. The reader has to be familiar with english anyways as the work is written in english as well.

Comment (guarin) [reply i18n/README.fr.md:377]: This is still outstanding

**Review by guarin — CHANGES_REQUESTED** [review]
The paper titles still need to be updated

**Pull request closed without merging**